misc/snapshotter/entrypoint.sh

```diff
@@ -17,10 +17,7 @@ if [ "$#" -eq 0 ]; then
 		--root ${NYDUS_LIB} \
 		--address ${NYDUS_RUN}/containerd-nydus-grpc.sock \
 		--log-level ${LEVEL} \
-		--enable-metrics=${ENABLE_METRICS} \
-		--enable-nydus-overlayfs=${ENABLE_NYDUS_OVERLAY} \
 		--daemon-mode ${NYDUSD_DAEMON_MODE} \
```

Comment:
Can we also remove `--daemon-mode` ? It seems using `shared`.

Reply:
`multiple` mode is the default one, so we should keep it?

Reply:
Ok, just keep `--daemon-mode` flag and use `multiple` by default.

Reply:
Let's change the shell variable default value to `multiple`

Reply:
done